### ISSUE TITLE
Check for Cached Item

### ISSUE THREAD

**Issue by ABausG** [kittinunf/Fuse#8]
Is there a way to check if an Item with a given Key is currently cached? Or in other words don't perform a network call when it is not cached?

**Comment by kittinunf** [comment]
The only way that you can do now is to ask via `get` interface. eg.

``` Kotlin
Fuse.jsonCache.get("foo") { result ->
  val (data, error) = result

  if (data != null) {
    //data is cached earlier 
  }
}
```

Yeah, I don't think this is convenience. I probably need to improve it somehow. Integrate with RxJava is one way to do just that. If you are willing to open a PR, I am happy to accept the improvement too!

**Comment by kittinunf** [comment]
Fixed in the new version! `1.0.0-alpha1`
